src/hooks/use-redirect-on-mount.ts

```diff
@@ -1,10 +1,10 @@
-import {useHistory, useLocation} from 'react-router';
-import {useOnMount} from './use-on-mount';
-import * as queryString from 'query-string';
-import {settSortering} from '../ducks/portefolje';
-import {useDispatch} from 'react-redux';
-import {erHeroku} from '../utils/utils';
-import {oppdaterValgtEnhet} from '../ducks/valgt-enhet';
+import { useHistory, useLocation } from "react-router";
+import { useOnMount } from "./use-on-mount";
+import * as queryString from "query-string";
+import { settSortering } from "../ducks/portefolje";
+import { useDispatch } from "react-redux";
+import { erGithubPages, erHeroku } from "../utils/utils";
+import { oppdaterValgtEnhet } from "../ducks/valgt-enhet";
```

Comment:
Good if you change your linting to the use of ' and no space on imports

Reply:
idk, that formatting style is by default...we will have more troubles if each one of us have personal preferences.